Tools/i18n/pygettext.py

```diff
@@ -1,11 +1,10 @@
 #! /usr/bin/env python3
-# -*- coding: iso-8859-1 -*-
```

Comment:
There's no other file that uses this encoding, I think it's safe (and more practical) to use utf-8.

Reply:
This is not related change, so please keep the coding cookie.

Reply:
Got it! I'll revert :) Would you accept a separate (perhaps not backported) PR that removes the coding and the commented-out code or do you think it's not worth it?

Reply:
I'll accept it if there are pygettext tests for files with non-UTF-8 encoding.

Reply:
Fair enough, I'll add it to my todo list :)